.ci/travis/openjdk14-test/jdk14-test-files.list

```diff
@@ -0,0 +1,2548 @@
+## Below input files are from
+##   https://download.java.net/openjdk/testresults/15/archives/24/langtools-24-summary.txt
```

Comment:
we need to keep copy of this file.
bash script do `sed` to get what you have.
after this we use our extra file to remove files that we are failing parsing for bug on our side. 

https://download.java.net/openjdk/testresults/15/  - is for jdk15 
so we 14 or 15 based ?
please add this link to comment as it might be useful in future.


and .... lets keep this file in "contribution" repo at https://github.com/checkstyle/contribution/tree/master/misc and we will 
ideally we should always down load it from java.net

Reply:
Done.